querydsl-collections/src/test/java/com/querydsl/collections/LeftJoinTest.java

```diff
@@ -62,7 +62,7 @@ public void List() {
     }
 
     @Test
-    public void Alias() {
+    public void alias1() {
```

Comment:
If something clashes, I'd say it's nice if we have a convention on how to 'escape' it.
Shall we use `1` or `_`? Or if you know something else?